ws/websocket.go

```diff
@@ -20,6 +20,31 @@ import (
 	"github.com/gorilla/websocket"
 )
 
+type Logger interface {
```

Comment:
I was about to suggest using the pre-existing `Logger` in `ocpp/types.go`, but then I realised this would add a dependency to the `ws` module, which should be completely agnostic.

What do you think about moving the existing `Logger` interface from `ocpp/types.go` to a separate module, to be re-used by the websockets as well? Mostly for easier/unified integration with existing loggers. Yes, you would lose that comfy `(id string, data []byte)` interface, so that's why I'm asking if you agree with this.

Reply:
I prefer the current interface since it allows customized behavior when logging the raw ocpp-j messages